lib/graphlient/errors/timeout_error.rb

```diff
@@ -1,6 +1,16 @@
 module Graphlient
   module Errors
     class TimeoutError < Error
+      def initialize(error, inner_exception = nil)
+        error_message = strip_inner_error_class_from_error_message(error)
+        super(error_message)
+
+        @inner_exception = inner_exception
+      end
+
+      def strip_inner_error_class_from_error_message(error)
```

Comment:
Is this really necessary? Maybe leave it alone?

Reply:
No worries.